routes/auth.js

```diff
@@ -52,7 +52,7 @@ module.exports = (passport) => {
 
   // Facebook will redirect the user to this URL after approval.
   router.get('/facebook/callback',
-    passport.authenticate('facebook', { successRedirect: '/',
-                                        failureRedirect: '/login'}));
+    passport.authenticate('facebook', { successRedirect: '/api/login/',
+                                        failureRedirect: '/api/error/'}));
```

Comment:
This is awesome but just not sure we want to have these under api

Reply:
yep probs true. Was unsure on these.

Reply:
Work fine for now, Might change this in a future PR